event_handler/sources.py

```diff
@@ -84,27 +84,25 @@ def get_source(headers):
     Gets the source from the User-Agent header
     """
     if "X-Gitlab-Event" in headers:
-        return "Gitlab"
+        return "gitlab"
 
-    if "Ce-Type" in headers and "tekton" in headers["Ce-Type"]:
-        return "Tekton"
+    if "tekton" in headers.get("Ce-Type", ""):
+        return "tekton"
 
-    if "User-Agent" in headers:
-        if "/" in headers["User-Agent"]:
-            return headers["User-Agent"].split("/")[0]
-        return headers["User-Agent"]
-
-    return None
+    if "GitHub" in headers.get("User-Agent", ""):
+        return "github"
+    
+    return "ERROR: UNKNOWN EVENT SOURCE"
 
 
 AUTHORIZED_SOURCES = {
-    "GitHub-Hookshot": EventSource(
+    "github": EventSource(
         "github", "X-Hub-Signature", github_verification
```

Comment:
I wonder if we need the nickname anymore.  Now that it's the key in the dictionary, this seems unnecessary.

Reply:
AFAICT, the nickname already wasn't being used. Either way, I removed it and will test that it works.